packages/@okta/vuepress-plugin-active-header-links/clientRootMixin.js

```diff
@@ -0,0 +1,51 @@
+/* global AHL_SIDEBAR_LINK_SELECTOR, AHL_HEADER_ANCHOR_SELECTOR, AHL_HEADER_TOP_OFFSET */
+
+import throttle from 'lodash.throttle'
+
+export default {
+  mounted () {
+    window.addEventListener('scroll', this.onScroll)
+  },
+
+  methods: {
+    onScroll: throttle(function () {
+      this.setActiveHash()
+    }, 300),
+
+    setActiveHash () {
+      const sidebarLinks = [].slice.call(document.querySelectorAll(AHL_SIDEBAR_LINK_SELECTOR))
+      const anchors = [].slice.call(document.querySelectorAll(AHL_HEADER_ANCHOR_SELECTOR))
+        .filter(anchor => sidebarLinks.some(sidebarLink => sidebarLink.hash === anchor.hash))
+
+      const scrollTop = Math.max(
+        window.pageYOffset,
+        document.documentElement.scrollTop,
+        document.body.scrollTop
+      )
+
+      for (let i = 0; i < anchors.length; i++) {
+        const anchor = anchors[i]
+        const nextAnchor = anchors[i + 1]
+
+        const isActive = i === 0 && scrollTop === 0
+          || (scrollTop >= anchor.parentElement.offsetTop - 40
```

Comment:
A comment or two here would be great

Reply:
This is the plugin code from vuepress.  Only hear until a PR is merged in